### PR TITLE
Added Per Jessen

### PR DESCRIPTION
**Pull request by pureooze**
Just testing to see if I did this right. Added Per Jessen, looks like it worked :)

### CODE REVIEW

File: sources/2011-04-04-per-jessen.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+
+**Star Trek or Star Wars?**
+
+I like both actually.
+
+
+**What is your favorite food and drink?**
+
+KÃ¤sespÃ¤tzle mit RÃ¶stzwiebel. Ittinger KlosterbrÃ¤u. Erdinger Dunkelweiss.
```
Comment (cornelius) [inline sources/2011-04-04-per-jessen.md]: There are some issues with non-ASCII characters here and at some other places in this file. I suppose this should rather be "Käsepätzle mit Röstzwiebel. Ittinger Klosterbräu. Erdinger Dunkelweiss".

**Comment by cornelius** [comment]
Nice. This works. There are some problems with non-ASCII characters, but they are already present in the original interviews as published on news.opensuse.org. Do you want to fix them in the pull request or should we do a later pass in cleaning things up after we have added the interviews?

**Comment by pureooze** [comment]
Yes, I will fix them up in the pull request, better do it now than put it off :)
I'll get to it over the weekend and will probably add more of the interviews too.

**Comment by pureooze** [comment]
I have updated the pages now, the non ASCII characters should be gone.

**Pull request merged**